.github/workflows/unit_test.yml

```diff
@@ -20,12 +20,12 @@ jobs:
   unit_tests:
     needs: cancel_previous_workflows
     runs-on: [self-hosted]
-    timeout-minutes: 60
+    timeout-minutes: 30
```

Comment:
unit_test speed a lot of time, consider add some time

Reply:
I have checked recent unit tests and no test will exceed 30 mins.